default_python/src/default_python/main.py

```diff
@@ -1,11 +1,10 @@
 from pyspark.sql import SparkSession
 
-def get_taxis():
-  spark = SparkSession.builder.getOrCreate()
+def get_taxis(spark: SparkSession):
   return spark.read.table("samples.nyctaxi.trips")
 
 def main():
-  get_taxis().show(5)
+  get_taxis(spark).show(5)
```

Comment:
So this depends on the global Spark. Does the linter accept that? And does this run when using DB Connect?

Reply:
Argh, you are right. This doesn't work with DBConnect. Only when running on the cluster. I'll take a look.

Reply:
I fixed that by explicitly use DBConnect from the main function